corehq/apps/users/bulkupload.py

```diff
@@ -487,12 +487,14 @@ def is_password(password):
                     if language:
                         user.language = language
                     if email:
-                        email_validator = EmailValidator()
                         try:
-                            email_validator(email)
-                            user.email = email
+                            validate_email(email)
                         except ValidationError:
                             raise UserUploadError(_("User has an invalid email address"))
+
+                        if not email.islower():
+                            raise UserUploadError(_("Email address must be lower case: {}".format(email)))
```

Comment:
Interesting. I'm still confused as to why we fail when there are capital letters in email addresses though (not why you are adding this, just why we are failing later down the line)